eos-paygd/eos-paygd.service.in

```diff
@@ -1,14 +1,22 @@
 [Unit]
 Description=Pay As You Go Daemon
 Documentation=man:eos-paygd(8)
-Before=systemd-user-sessions.service
+DefaultDependencies=no
```

Comment:
Please add a comment to explain why DefaultDependencies=no

Reply:
Added a comment